Typos and stuff

renhiyama opened the pull request:
Eval page

es183923 commented on docpages/03_example_programs.md:
```diff
@@ -1637,11 +1637,11 @@ The above is just a very simple example. You can also send embed messages. All y
 Needless to say, this is very dangerous. If you are asking how to do this, and want to put this into your bot, we trust that you have a very good reason to do so and have considered alternatives before resorting to this. The code below is for educational purposes only and makes several assumptions:
 
-1. This code will only operate on UNUX-like systems such as Linux
+1. This code will only operate on UNIX-like systems such as Linux (doesn't include Darwin, like Mac OS though)
```
"Mac OS" should be "macOS"

renhiyama replied:
I dont see a difference between them. Also it's better to pronounce Mac OS and not macOS

es183923 replied:
it is supposed to be stylized as "macOS" not"Mac OS".

renhiyama replied:
Dpp doesn't even focus on Mac, and neither do I. Anyways mac os is a shit, and I don't want to focus a lot of that simple spacing in their name.

renhiyama replied:
Dpp doesn't even focus on Mac, and neither do I. Anyways mac os is a shit, and I don't want to focus a lot of that simple spacing in their name.

braindigitalis commented:
any progress on this one? I think only the last chunk is needed as it can run on osx with changes, and the pun at the top would be better kept.

es183923 commented:
> any progress on this one? I think only the last chunk is needed as it can run on osx with changes, and the pun at the top would be better kept.

It doesn't work on macOS, because `link.h` and `dlfcn.h` do not exist there.

es183923 commented on docpages/03_example_programs.md:
```diff
@@ -1656,6 +1656,7 @@ The evaluated code will run within its own thread, so can execute for as long as
 This code operates by outputting your provided code to be evaluated into a simple boilerplate program which can be compiled to a
 shared object library (.so file). This .so file is then compiled with g++, using the `-shared` and `-fPIC` flags. If the program can be successfully compiled, it is then loaded using `dlopen()`, and the symbol `so_exec()` searched for within it, and called. This `so_exec()` function will contain the body of the code given to the eval command. Once this has been called and it has returned,
 the `dlclose()` function is called to unload the library, and finally any temporary files (such as the .so file and its corresponding .cpp file) are cleaned up.
+Docker is definitely recommended if you code on Windows/Mac OS, because docker desktop still uses a linux VM, so your code can easily use `.so` file and your code runs the same on your vps (if it also uses Linux distro)
```
@braindigitalis Can you give your opinion on this?

renhiyama replied:
See down

renhiyama commented:
@es183923 use docker, since docker desktop runs a linux vm, so your code can easily use .so file. Just added that information in the new commit.
Also @braindigitalis (oh wait, you had this username on GitHub Brain lol? I thought it was some random guy) I added your pun back, along with tip/suggestion. I can argue about using eval, and they aren't evil, the whole day, so better you keep that part after the pun lol
